sdks/java/io/clickhouse/build.gradle

```diff
@@ -49,7 +49,7 @@ idea {
   }
 }
 
-def clickhouse_jdbc_version = "0.2.6"
+def clickhouse_jdbc_version = "0.3.2-patch10"
```

Comment:
Why not to the latest version (`0.3.2-patch11`)?

Reply:
Cause some conflict with testcontainers_version

Reply:
Sounds good